hack/tests/e2e-ansible-molecule.sh

```diff
@@ -65,7 +65,7 @@ header_text "Test Ansible Molecule scenarios"
 pushd "${ROOTDIR}/test/ansible"
 DEST_IMAGE="quay.io/example/ansible-test-operator:v0.0.1"
 sed -i".bak" -E -e 's/(FROM quay.io\/operator-framework\/ansible-operator)(:.*)?/\1:dev/g' build/Dockerfile; rm -f build/Dockerfile.bak
-operator-sdk build "$DEST_IMAGE"
+operator-sdk build "$DEST_IMAGE" --image-build-args="--no-cache"
```

Comment:
I can break this into a different PR if requested or controversial.

Reply:
I'm fine with keeping it if it made the tests pass :)